xdebug.c

```diff
@@ -2575,12 +2610,16 @@ ZEND_DLEXPORT void xdebug_statement_call(zend_op_array *op_array)
 	if (xdebug_is_debug_connection_active_for_current_pid()) {
 
 		if (XG(context).do_break) {
+			block=1;
 			XG(context).do_break = 0;
 
 			if (!XG(context).handler->remote_breakpoint(&(XG(context)), XG(stack), file, lineno, XDEBUG_BREAK, NULL, 0, NULL)) {
 				xdebug_mark_debug_connection_not_active();
-				return;
 			}
+			if (xdebug_is_debug_connection_active_for_current_pid()) {
+                XG(context).handler->cmdloop(&(XG(context)), 1 TSRMLS_CC, block);
+            }
+            block =0;
```

Comment:
This seems to use spaces, and not tabs. There are also missing spaces around `=` in the three new `block` variable usages.